src/Symfony/Component/ExpressionLanguage/SerializedParsedExpression.php

```diff
@@ -12,7 +12,7 @@
 namespace Symfony\Component\ExpressionLanguage;
 
 /**
- * Represents an already parsed expression.
+ * Represents an already serialized parsed expression.
```

Comment:
Not sure about this one, but we already have `ParsedExpression` with description "Represents an already parsed expression."